src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -212,6 +212,14 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                         </label>
                     </div>
                 </div>
+                <div class="form-group-narrow" *ngIf="visibilityCheck()">
```

Comment:
visibilityCheck is not a good method name here

Reply:
i renamed the method. Now it is called isInProduction